website/docs/components/template/index.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Template
+hidden: true
+description: {Optional long description that appears in the cover}
+caption: {Short description that appears in the cards}
+status: released
+links:
+  figma: https://www.figma.com/file/noyY6dUMDYjmySpHcMjhkN/HDS-Product---Components?t=nsKJzzKtdX5A7zpz-7
+  github: https://github.com/hashicorp/design-system/tree/main/packages/components/addon/components/hds
+---
+
+<section data-tab="Guidelines">
+  @include "partials/guidelines/overview.md"
+  @include "partials/guidelines/guidelines.md"
+  @include "partials/guidelines/related.md"
```

Comment:
Since this is going to come from the frontmatter I don't think we will want to have this here as part of a template

Reply:
Ok, that's fair. I'll update the template and the `alert` documentation as well.